components/automate-ui/src/app/pages/notification-form/notification-form.component.html

```diff
@@ -69,6 +69,14 @@
               {{model.rule.AlertTypeLabels[key]}}
             </chef-option>
           </chef-select>
+          <div *ngIf="displayCriticalControlsCheckbox()">
+            <p class="small"><em>Check to be notified of critical control failures only</em>&nbsp;
```

Comment:
Can we move the checkbox over to the left please, and use the label `Only send a notification for critical control failures`

Reply:
![Screenshot 2019-05-23 at 11 46 16](https://user-images.githubusercontent.com/34461777/58247145-d8d1e200-7d50-11e9-831b-15c6d3d3670b.png)

I also verified that you can tab between all the fields and check/uncheck with the space bar.